week-2/D-predicates/exercise.js

```diff
@@ -2,16 +2,19 @@
   Predicates
   ---------------------------------
   Write a predicate to predicates
-  The variables should have values that match the expected results.
+  The variables should have vcd .calues that match the expected results.
 */
 
 // Finish the predicate function to test if the passed number is negative (less than zero)
-function isNegative(number) {
 
+function isNegative(number) {
+ // return typeOf number === "number";
+    return number<0;
 }
 
 // Finish the predicate function to test if the passed number is between 0 and 10
-function isBetweenZeroAnd10(number) {
+  function isBetweenZeroAnd10(number) {
+  var number = 0
```

Comment:
This function should `return` either `true` or `false` depending on whether `number` is between zero and 10 or not.

You can use an if statement to check if `number` passes the condition. 

``` 
if (condition) { 
  return true;
} else {
  return false;
}
```

What should the condition be?

There are actually two conditions: 
- bigger than or equal to zero
- less than or equal to 10

Both those conditions should be true. What logical operator can you use for that?